### ISSUE TITLE
Scorch fails on samples with no test clusters

### ISSUE THREAD

**Issue by Rexhaif** [LoicGrobol/scorch#9]
I've tried scorch and found, that it fails on certain documents from conll-2012 test set, specifically:
- bc_cnn_00_cnn_0008-008.json
- nw_wsj_23_wsj_2390-000.json
It seems like these documents have no gold clusters, and scorch fails with division by zero exception:
```
Traceback (most recent call last):                                                                                                         
  File "/usr/local/bin/scorch", line 8, in <module>
    sys.exit(main_entry_point())
  File "/usr/local/lib/python3.8/dist-packages/scorch/main.py", line 240, in main_entry_point
    out_stream.writelines(process_dirs(gold_path, sys_path))
  File "/usr/local/lib/python3.8/dist-packages/scorch/main.py", line 200, in process_dirs
    r = {
  File "/usr/local/lib/python3.8/dist-packages/scorch/main.py", line 201, in <dictcomp>
    name: metric(gold_clusters, sys_clusters)
  File "/usr/local/lib/python3.8/dist-packages/scorch/scores.py", line 118, in b_cubed
    R = math.fsum(
ZeroDivisionError: float division by zero
```

**Comment by LoicGrobol** [comment]
Oh thanks, sorry for that. Just to be clear: you have converted the gold conll files and some system output our yours using <https://github.com/LoicGrobol/scorch/blob/master/scorch/conll.py> and if fails when you try to score them with scorch, is that right?

**Comment by Rexhaif** [comment]
Yes, actually I was just testing how scorch works, so I've converted conll-2012 test set using conll.py and evaluated it against himself, like this:
```
scorch data/gold data/gold
```

**Comment by Rexhaif** [comment]
> Yes, actually I was just testing how scorch works, so I've converted conll-2012 test set using conll.py and evaluated it against himself, like this:
> 
> ```
> scorch data/gold data/gold
> ```
And then I've written bash for loop to evaluate every file, to find which files causes exception

**Comment by LoicGrobol** [comment]
Fixed by #10
